addons/account/views/account_move_views.xml

```diff
@@ -1465,6 +1473,9 @@
                     <filter name="posted" string="Posted" domain="[('state', '=', 'posted')]"/>
                     <filter name="cancel" string="Cancelled" domain="[('state', '=', 'cancel')]"/>
                     <separator/>
+                    <filter name="sent" string="Sent" domain="[('is_move_sent', '=', True)]" invisible="context.get('default_move_type') in ('in_invoice', 'in_refund', 'in_receipt')"/>
```

Comment:
And while we are at it (i should have told you sooner oooppss) we can do the same than for move sent values. I think that the limit is in general 120 characters before moving it on other multiple lines :smile:

Reply:
That's a good rule of thumb I'll keep in mind :) thanks!